_data/simple-icons.json

```diff
@@ -4346,7 +4348,8 @@
         {
             "title": "Ingress",
             "hex": "783CBD",
-            "source": "https://ingress.com/assets/fonts/ingress_icons.woff"
+            "source": "https://niantic.helpshift.com/a/ingress/?s=ingress-events&f=brand-and-fan-site-guidelines&p=web",
```

Comment:
The font used as source is available on their website:
https://ingress.com/

Reply:
I will reverse this one. Notice that I personally have no problem with PNG sources for geometrical logos such as these, and I prefer them to fonts.
This is being discussed in other comment threads, tho. I will resolve this after reversing to status quo ante.

Reply:
Fonts includes vector versions. It is always better than a Raster format. Personally, I have a problem with PNG when there are better source.